modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/TokensAutodetector/ImportToken.swift

```diff
@@ -114,6 +109,11 @@ final public class ImportToken: TokenImportable, TokenOrContractFetchable {
         }.then(on: queue, { [queue] contract -> Promise<TokenOrContract> in
             let key = "\(contract.hashValue)-\(onlyIfThereIsABalance)-\(server)"
 
+            //Useful to check because we are/might action-only TokenScripts for native crypto currency
```

Comment:
Works, but should change the commit message though. We fixed the code, not just the test suite

Reply:
tests are still fail, trying to figure it out